lab#1/task#3/build-client.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# 1. Install app deps
+npm install
+echo "Modules installed"
+
+# 2. Set ENV variable for build configuration
+export ENV_CONFIGURATION=production
+echo "Set ENV_CONFIGURATION variable"
+
+# 3. Run build
+# npm run build -- --configuration $ENV_CONFIGURATION
+# NOTE: command result in error
+echo "Build created (skipped)"
```

Comment:
I am not sure what's the problem on your side, but this command worked on mine

<img width="921" alt="Screenshot 2022-09-19 at 18 56 21" src="https://user-images.githubusercontent.com/26553426/191060608-3e6e8f50-5697-47f5-98f1-d2357f2adbdd.png">